client/misc.go

```diff
@@ -1,21 +1,29 @@
 package client
 
+import "regexp"
+
 func GetSchedule(schedule string) (typefmt string, cronfmt string) {
 	var TypeStr string
 	var CronStr string
 
 	switch schedule {
```

Comment:
Should we do a `strings.ToLower(schedule)` to catch everything?

Reply:
Have implemented this.